chain/default_syncer.go

```diff
@@ -383,6 +389,9 @@ func (syncer *DefaultSyncer) HandleNewTipset(ctx context.Context, tipsetCids typ
 			syncer.badTipSets.AddChain(chain[i:])
 			return err
 		}
+		if (i % 500) == 0 {
+			logSyncer.Infof("processing block %d of %v", i, highestBlock)
```

Comment:
Blocking: this is misleading because it is conflating height and number of blocks which are not the same.  There are a few different ways you can do this.  One way would be "processing block x of y for chain with head at c" where y is the length of the fetched chain (`len(chain)`), x is `i`, and c is the `tipsetCids` above.

You could also do this strictly by height "processing block height u of v" where v is the highest block as now and u is the height of tipset at index i.